server/src/apub/mod.rs

```diff
@@ -112,15 +110,27 @@ fn is_apub_id_valid(apub_id: &Url) -> bool {
     .map(|d| d.to_string())
     .collect();
   match apub_id.domain() {
-    Some(d) => allowed_instances.contains(&d.to_owned()),
-    None => false,
+    Some(d) => {
+      let contains = allowed_instances.contains(&d.to_owned());
+
+      if !contains {
+        debug!("{} not in {:?}", d, allowed_instances);
+      }
```

Comment:
Thanks for adding this, it'll def help. (I imagine it did due to lemmy_alpha changing to lemmy-alpha)